Test/Unit/Visitor/Arithmetic.php

```diff
@@ -101,4 +99,92 @@ public function case_visitor_exhaustively()
                 }
             });
     }
+
+    public function case_visitor_variable()
+    {
+        $this
+            ->given(
+                $compiler      = Compiler\Llk\Llk::load(new File\Read('hoa://Library/Math/Arithmetic.pp')),
+                $visitor       = new CUT(),
+                $variableName  = uniqid('v'),
+                $variableValue = rand(0, PHP_INT_MAX)
+            )
+            ->then
+                ->object($compiler->parse($variableName . ' * 2'))->isInstanceOf('Hoa\Compiler\Llk\TreeNode')
```

Comment:
We just check the compiler parses the expression correctly and ensure that there is no regression on variables.

If this fails, no need to go further in the test.